docs/snippets/all/archetypes/image_encoded.cpp

```diff
@@ -0,0 +1,20 @@
+// Create and log a image.
+
+#include <rerun.hpp>
+
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+namespace fs = std::filesystem;
+
+int main() {
+    const auto rec = rerun::RecordingStream("rerun_example_image_encoded");
+    rec.spawn().exit_on_failure();
+
+    fs::path image_filepath = fs::path(__FILE__).parent_path() /
+                              "../../../../crates/viewer/re_ui/data/logo_dark_mode.png";
```

Comment:
This looks pretty ugly in the docs & code comments. I think it would be worth the cost having an example image (or symlink!) directly in the snippet folder here.
Is an absolute path needed?

Reply:
No symlinks, they are super annoying in git, especially when moving files.

I'll add a `ferris.png` in the same folder instead.

And yeah, I suggest absolute paths are needed, because who knows from what folder these snippets are run from.